service/history/transferQueueActiveProcessor.go

```diff
@@ -403,7 +400,7 @@ func (t *transferQueueActiveProcessorImpl) processCloseExecution(task *persisten
 
 	workflowTypeName := executionInfo.WorkflowTypeName
 	workflowStartTimestamp := executionInfo.StartTimestamp.UnixNano()
-	workflowCloseTimestamp := msBuilder.GetLastUpdatedTimestamp()
```

Comment:
Can you remove GetLastUpdatedTimestamp API from mutableState.  Looks like this is used also on DescribeWorkflowExecution and will now give incorrect information.

Reply:
Removed